conftest.py

```diff
@@ -61,3 +70,10 @@ def release_calendar_page(release_calendar_index) -> ReleaseCalendarPage:  # pyl
     page.save()
 
     return page
+
+
+@pytest.fixture()
+def document():
+    """Creates a test document."""
+    file = ContentFile(b"A boring example document", name="file.txt")
```

Comment:
Nit: Text files are text
```suggestion
    file = ContentFile("A boring example document", name="file.txt")
```